types/index.d.ts

```diff
@@ -107,6 +122,28 @@ export class Keypair {
   xdrMuxedAccount(id: string): xdr.MuxedAccount;
 }
 
+export const LiquidityPoolFeeV18 = 30;
+
+export function getLiquidityPoolId(liquidityPoolType: LiquidityPoolType, liquidityPoolParameters: LiquidityPoolParameters): Buffer;
+
+export function validateLexicographicAssetsOrder(assetA: Asset, assetB: Asset): boolean;
```

Comment:
I would be tempted to do something like:
`Asset.isSorted(assets ...Asset): boolean`, and something like a `Asset.compare(a: Asset, b: Asset): number` so you could do `[a, b].sortBy(Asset.compare)` as well.

Maybe that's overengineering it... wdyt?

Reply:
`Asset.compare` is a much better place for that kind of logic, thank you! 
I've updated that in https://github.com/stellar/js-stellar-base/commit/11537633dfa7eed32f92931418165d23882fd56b.

I don't think we need `Asset.isSorted` though, `Asset.compare` should be enough since it's a common pattern in javascript objects.